### ISSUE TITLE
Multi Item Imports vs Duplicates "froms"

### ISSUE THREAD

**Issue by cancan101** [4Catalyzer/python#18]
What do people think about these two import styles:
```python
from project.stuff import foo
from project.stuff import bar
```
vs.
```python
from project.stuff import foo, bar
# or
from project.stuff import (
    foo,
    bar,
)
```
/CC @nathansilberman @taion @itajaja @nevsan

**Comment by itajaja** [comment]
second preferable

**Comment by taion** [comment]
One `import` statement per module please. See https://www.python.org/dev/peps/pep-0008/#imports.

**Comment by nevsan** [comment]
Yes. And we should remind people that you can use parentheses to break them into multiple lines if they exceed the line length (any opinions here? I don't think it's necessary to always use parentheses if it fits on one legal line length).

**Comment by nathansilberman** [comment]
I like the first one, but it seems I'm clearly in the minority. For option #2, if we cannot fit the list on one line, which is preferrable:

Option a:

from a.b.c import (
  d, e, f,
)

or Option b:

from a.b..c import (
  d,
  e, 
  f,
)

My pref is option (a)

**Comment by nevsan** [comment]
I like option (a). YAPF does something more like option (a). I usually prefer the YAPF default decisions when I don't have strong opinions.

**Comment by taion** [comment]
I have something like this in the multiline section. The best way to go is to fit everything on one line if possible; otherwise, to put everything on its own line, per https://github.com/4Catalyzer/flask-resty/pull/136#discussion_r129663331.

So

```python
# When everything fits on one line:
(
    a, b, c,
)

# Assume these don't fit on one line:
(
    long_identifier_1,
    long_identifier_2,
    long_identifier_3,
)

# But almost never:
(
    a,
    b, c,
)
```

The latter avoids cruft in diffs from re-wrapping lines when e.g. alphabetizing imports.

There are very rare exceptions to the last; e.g. SQLAlchemy has `Index(name, *columns)`, but:

```python
# This:
Index(
    'ix_name',
    column_1, column_2, column_3,
)

# Is better than:
Index(
    'ix_name',
    column_1,
    column_2,
    column_3,
)
```
